storage/simulation/cluster.go

```diff
@@ -176,3 +251,36 @@ func (c *Cluster) String() string {
 
 	return buf.String()
 }
+
+// StringEpochHeader creates the string header for epoch outputs based on all
+// of the current stores.
+func (c *Cluster) StringEpochHeader() string {
+	var buf bytes.Buffer
+	buf.WriteString("Store:\t")
+	for _, storeID := range c.storeIDs {
+		fmt.Fprintf(&buf, "%d\t", storeID)
+	}
+	return buf.String()
+}
+
+// StringEpoch create a string with the current free capacity for all stores.
+func (c *Cluster) StringEpoch() string {
+	var buf bytes.Buffer
```

Comment:
if you decide to move the equivalent declaration in my other comment, move this one too.